etherman/etherman.go

```diff
@@ -1368,8 +1368,10 @@ func decodeSequencedBatches(smcAbi abi.ABI, txData []byte, forkID uint64, lastBa
 		var hashes []common.Hash
 		for i, validiumData := range sequencesValidium {
 			bn := lastBatchNumber - uint64(len(sequencesValidium)-(i+1))
-			batchNums = append(batchNums, bn)
-			hashes = append(hashes, validiumData.TransactionsHash)
+			if validiumData.ForcedTimestamp == 0 {
+				batchNums = append(batchNums, bn)
+				hashes = append(hashes, validiumData.TransactionsHash)
+			}
```

Comment:
```suggestion
			} else {
			    forcedBatchNums = append(forcedBatchNums, bn)
			}
			// ...
		forcedBatchL2Data, err := GetForcedBatchL2Data(batchNums)
```

We still need to get the data for forced batches, it's just that this data needs to be found on the forced bnatches table instead of the DA layer